src/api/package.js

```diff
@@ -64,11 +75,16 @@ export default async (providedOptions = {}) => {
   const forgeConfig = await getForgeConfig(dir);
   let packagerSpinner;
 
-  const packageOpts = Object.assign({
-    asar: false,
-    overwrite: true,
-  }, forgeConfig.electronPackagerConfig, {
-    afterCopy: [async (buildPath, electronVersion, pPlatform, pArch, done) => {
+  const pruneEnabled = !('prune' in forgeConfig.electronPackagerConfig) || forgeConfig.electronPackagerConfig.prune;
```

Comment:
This seems overly complicated, why is the `in` check required?

Reply:
Because if prune doesn't exist, the default is true.